lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -60,22 +62,15 @@ public void TestTaskCloseOnLocalRuntime()
             const int numTasks = 4;
             var testFolder = DefaultRuntimeFolder + TestId;
             TestBroadCastAndReduce(false, numTasks, chunkSize, dims, iterations, mapperMemory, updateTaskMemory, testFolder);
-            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder);
-            var failedCount = GetMessageCount(lines, FailTaskMessage);
+            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder, 120);
```

Comment:
@markusweimer  @tcNickolas  what do you guys think? Is it ok to have a parameter like 200 here or do we need to create our own Map and Update task here where in each iteration they sleep for a second or so so that we get a good timing control.